srcpkgs/ly/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'ly'
+pkgname=ly
+version=0.1
+revision=2
+build_style=gnu-makefile
+make_use_env=yes
+makedepends="libX11-devel pam-devel ncurses-devel"
+depends="ncurses pam util-linux xauth xinit"
+short_desc="Ncurses display manager"
+maintainer="travankor <travankor@tuta.io>"
+license="WTFPL"
+homepage="https://github.com/cylgom/ly"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=46db2e265a44bbed0242980c3935d495035a0033afb37c265e179c57b032d69e
+
+do_configure() {
+	sed -i -e "/CXX := cc/d" \
+	-e 's/$(CXX)/$(CC)/' \
+	-e "s/CXXFLAGS/CFLAGS/" \
+	-e "s/CFLAGS :=/CFLAGS +=/" \
+	-e 's/LDDFLAGS := $(LIBS)/LDDFLAGS := $(LDFLAGS) $(LIBS)/' \
```

Comment:
Sorry @travankor I mixed this up. `$(LIBS)` has to come before `$(LDFLAGS)`.

So this line should be
`-e 's/LDDFLAGS := $(LIBS)/LDDFLAGS := $(LIBS) $(LDFLAGS)/' \`